portfolio/Makefile

```diff
@@ -0,0 +1,11 @@
+install:
+	npm install --no-package-lock --no-audit
+	echo "#!/bin/sh\n\tcd portfolio && make format && make lint" > ../.git/hooks/pre-push
+	chmod +x ../.git/hooks/pre-push
+
+format: install
+	npx prettier --write --ignore-path=target --ignore-path=src/main/webapp/node_modules --ignore-path=src/main/webapp/dist src/main/webapp/**/*.{html,css,js}
```

Comment:
I added extra --ignore-path and --ignore-pattern arguments to prevent extraneous linting errors.